src/Props/Container.php

```diff
@@ -102,7 +102,7 @@ public function __isset($name)
     /**
      * {@inheritdoc}
      */
-    public function has($name)
+    public function has($name): bool
```

Comment:
is this backward compatible with psr/container 1.1.2?

- https://github.com/mrclay/minify/issues/703